products/yocto.md

```diff
@@ -16,15 +16,22 @@ changelogTemplate: |
 iconSlug: NA
 releaseLabel: "__RELEASE_CYCLE__ '__CODENAME__'"
 releases:
+-   releaseCycle: "4.1"
+    codename: 'Langdale'
+    latest: "4.1.1"
+    eol: 2023-03-27
```

Comment:
According to https://wiki.yoctoproject.org/wiki/Releases, 4.1 will be supported until May 2023 (no day specified) but you put 2023-03-27. Between these two dates, which is the correct one?

Reply:
Good catch. I made a typo 😄